api/aether-top-level-openapi3.yaml

```diff
@@ -134,76 +134,265 @@ components:
     Transaction:
       description: Transaction refers to a multi-target transactional change. Taken from https://github.com/onosproject/onos-api/tree/master/proto/onos/config/v2
       properties:
+        meta:
+          description:
+          properties:
+            key:
+              description:
+              type: string
+            version:
+              description:
+              type: integer
+              format: uint64
+            revision:
+              description: the change revision number
+              properties:
+                revision:
+                  description: Revision is a revision number
+                  type: integer
+                  format: int64
+            created:
+              description: the time at which the transaction was created
+              type: string
+              format: date-time
+            updated:
+              description: the time at which the transaction was last updated
+              type: string
+              format: date-time
+            deleted:
+              description: a flag indicating whether this transaction is being deleted by a snapshot
```

Comment:
I think it's the **time** it's being deleted at

Reply:
done